src/Timer.ts

```diff
@@ -13,7 +14,7 @@ class Timer<T = void>
    * Delay in milliseconds
    * This may be `Infinity`
    */
-  public readonly delay: number;
+  protected delay_: number;
```

Comment:
Was there a reason to change these properties?

Reply:
They're updated outside of the constructor now, So I can't use readonly. I made it protected and access it with a getter now.